src/dsql/parse.y

```diff
@@ -703,6 +703,9 @@ using namespace Firebird;
 %token <metaNamePtr> CALL
 %token <metaNamePtr> FORMAT
 %token <metaNamePtr> NAMED_ARG_ASSIGN
+%token <metaNamePtr> BTRIM
```

Comment:
Reorder it alphabetically inside block `// tokens added for Firebird 6.0`, like the others.